src/js/components/githubIssues/issueCard.tsx

```diff
@@ -0,0 +1,88 @@
+import Card from '@salesforce/design-system-react/components/card';
+import Dropdown from '@salesforce/design-system-react/components/menu-dropdown';
+import i18n from 'i18next';
+import React, { useCallback, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { GitHubIssueLink } from '@/js/components/githubIssues/selectIssueModal';
+import { SpinnerWrapper, useFetchIssueIfMissing } from '@/js/components/utils';
+import { ThunkDispatch } from '@/js/store';
+import { updateObject } from '@/js/store/actions';
+import { OBJECT_TYPES } from '@/js/utils/constants';
+
+interface Props {
+  issueId: string;
+  epicId?: string;
+  taskId?: string;
+}
+
+const IssueCard = ({ issueId, epicId, taskId }: Props) => {
+  const dispatch = useDispatch<ThunkDispatch>();
+  const { issue } = useFetchIssueIfMissing({ id: issueId });
+
+  const unlink = useCallback(() => {
+    if (epicId) {
+      dispatch(
+        updateObject({
+          objectType: OBJECT_TYPES.EPIC,
+          url: window.api_urls.epic_detail(epicId),
+          data: { issue: null },
+          patch: true,
+        }),
+      );
+    } else if (taskId) {
+      dispatch(
+        updateObject({
+          objectType: OBJECT_TYPES.TASK,
+          url: window.api_urls.task_detail(taskId),
+          data: { issue: null },
+          patch: true,
+        }),
+      );
+    }
+  }, [dispatch, epicId, taskId]);
+
+  useEffect(() => {
+    if (issue === null) {
+      unlink();
+    }
+  }, [issue, unlink]);
+
+  return (
+    <Card
+      bodyClassName="slds-card__body_inner"
+      heading={
+        issue
+          ? `#${issue.number}: ${issue.title}`
+          : i18n.t('Loading GitHub Issue…')
+      }
```

Comment:
@stacyk (cc @SondraE) Right now, because the title of the GitHub issue is the heading of the <Card /> component, if the title is long it gets truncated (but full text available on hover):

![Screen Shot 2021-11-19 at 4 32 57 PM](https://user-images.githubusercontent.com/552316/142694898-11633cb4-48b5-42b3-b7b2-bda2ad99bf11.jpg)

![Screen Shot 2021-11-19 at 4 32 53 PM](https://user-images.githubusercontent.com/552316/142694900-14392482-c358-492f-b8a2-f98d4c40f5be.jpg)

I think that's probably okay? Most titles will probably not be extremely long.

But if we prefer, either @stacyk could try to find a way to _not_ truncate them, or we could move the title into the Card body:

![Screen Shot 2021-11-19 at 4 32 17 PM](https://user-images.githubusercontent.com/552316/142694978-279bad84-138a-41a1-af50-b606eec6a6e8.jpg)